package lesson04.part03;
//@FoLoKe
import org.junit.Test;
import org.junit.runner.RunWith;
import org.junit.runners.JUnit4;

import static org.junit.Assert.*;

@RunWith(JUnit4.class)
public class Task41Test {
    /* 1.	Программа должна выводить сообщение на экран.
     * 2.	В программе должен быть блок try-catch.
     * 3.	Программа должна отлавливать исключения конкретного типа, а не все возможные (Exception).
     * 4.	Выведенное сообщение должно содержать тип возникшего исключения.
     * 5.	Имеющийся код в методе main не удалять.
     */

    @Test
    public void test41main() {

    }

    @Test
    public void test4101() {

    }

    @Test
    public void test4102() {

    }

    @Test
    public void test4103() {

    }

    @Test
    public void test4104() {

    }
}